### PR TITLE
Replace openvino_contrib with openvino_tokenizers

### PR DESCRIPTION
**Pull request by Wovchena**
Close https://github.com/openvinotoolkit/openvino.genai/issues/228

### CODE REVIEW

File: .github/workflows/causal_lm_cpp.yml [hunk 1]
```diff
@@ -95,22 +95,6 @@ jobs:
...
-          tokenizer = transformers.LlamaTokenizer.from_pretrained('TinyLlama/TinyLlama-1.1B-Chat-v1.0')
-          tokenized = tokenizer('return 0', return_tensors='pt')
-          for beam in transformers.LlamaForCausalLM.from_pretrained('TinyLlama/TinyLlama-1.1B-Chat-v1.0').generate(**tokenized, num_beam_groups=3, num_beams=15, num_return_sequences=15, diversity_penalty=1.0, max_new_tokens=20, early_stopping=False, length_penalty=1.0, no_repeat_ngram_size=9**9, do_sample=False):
-              ref = ': ' + tokenizer.decode(beam[tokenized['input_ids'].numel():], skip_special_tokens=True) + '\n'
-              idx = predictions.find(ref)
-              if -1 == idx:
-                  raise RuntimeError(f'Missing "{ref=}" from predictions')
-              predictions = predictions[:idx] + predictions[idx + len(ref):]
-          "
-          echo return 0 passed
```
Comment (ilya-lavrenov) [inline .github/workflows/causal_lm_cpp.yml:112]: why has it been removed?

Comment (Wovchena) [reply .github/workflows/causal_lm_cpp.yml:112]: newer transformers modified beam search

Comment (ilya-lavrenov) [reply .github/workflows/causal_lm_cpp.yml:112]: do we plan to align with their impl separately?

**Review by ilya-lavrenov — APPROVED** [review]

File: .github/workflows/causal_lm_cpp.yml (hunk 1 above)
Comment (Wovchena) [reply .github/workflows/causal_lm_cpp.yml:112]: Yes. You can find it in the list of tasks I've send
> 1. d. Align group beam search with new transformers version

**Pull request merged**